lib/datadog/tracing/contrib/span_attribute_schema.rb

```diff
@@ -0,0 +1,25 @@
+module Datadog
+  module Tracing
+    module Contrib
+      # Contains methods for fetching values according to span attributes schema
+      module SpanAttributeSchema
+        module_function
+
+        def fetch_service_name(env, default)
+          ENV.fetch(env) do
+            if Datadog.configuration.tracing.span_attribute_schema == 'v1'
```

Comment:
I recommend moving `v1` to a constant, alongside `SpanAttributeSchema::ENV_DEFAULT_SCHEMA` given it represents a value the is exposed to the outside world.